html/changelogs/TheLion1675 - Spellcheck.yml

```diff
@@ -34,3 +34,4 @@ delete-after: True
 # Please surround your changes in  double quotes ("), as certain characters otherwise screws up compiling. The quotes will not show up in the changelog.
 changes: 
   - spellcheck: "Fixed a spelling error in Containment round description."
```

Comment:
It's bad form to edit changelogs created by other PRs, they may not be existing by the time this is merged

Reply:
I'll pull it over.

Reply:
Fixed.